test/parallel/test-cli-node-options-docs.js

```diff
@@ -123,6 +123,20 @@ for (const [, envVar, config] of nodeOptionsCC.matchAll(addOptionRE)) {
   }
 }
 
+{
+
+  const start = /^## Options/m;
+  const end = /^## Environment variables/m;
+  const filteredCLIText = cliText.slice(cliText.search(start), cliText.search(end)).trim();
```

Comment:
The trimming is not useful, is it?
```suggestion
  const filteredCLIText = cliText.slice(cliText.search(start), cliText.search(end));
```